server/src/main/java/io/littlehorse/common/ServerContext.java

```diff
@@ -0,0 +1,17 @@
+package io.littlehorse.common;
+
+import io.littlehorse.common.model.getable.global.acl.PrincipalModel;
+
+public interface ServerContext {
+
+    enum Scope {
```

Comment:
Can you please put a comment here so that someone who doesn't understand the difference between GRPC Server and Streams Command Processor can understand this? It makes 100% sense to me but probably not for @diablouma ... when he does his first LH Server ticket he'd probably get confused :joy: